app/controllers/admin/api/policies_controller.rb

```diff
@@ -11,9 +11,9 @@ class Admin::Api::PoliciesController < Admin::Api::BaseController
   #
   ##~ op             = e.operations.add
   ##~ op.httpMethod  = "GET"
-  ##~ op.summary     = "APIcast Policies Registry"
+  ##~ op.summary     = "Gateway Policies Registry"
   ##~ op.description = "Returns APIcast Policies Registry"
-  ##~ op.group = "apicast_policies"
+  ##~ op.group = "gateway_policies"
```

Comment:
So we had a discussion about it with @guicassolato. Not related to this PR but to understand where we are going.

We were wondering if we should separate each API by role/scope. Meaning basically that if you are a member only having access to `finance_api` for example, you should not be able to operate `account_management_api` or even see the APIs spec.

Does it make sense in the long term @3scale/system @3scale/product ?

Reply:
So we will be renaming from now on to make it less tied to APIcast and more open to other proxies, it seems.. right?

Reply:
Given in this case, Policies are something unique from APIcast